content/posts/2025/04/maintainer-onboarding-2025.md

```diff
@@ -0,0 +1,69 @@
+Read on for details!
+
+## Who can be a Maintainer?
+
+You do not need to be a Git expert to be a Maintainer. 
+If you are familiar with the current [Carpentries Git materials](https://swcarpentry.github.io/git-novice/), you will have the foundation and support to learn [the skills you will need as a Maintainer](https://www.youtube.com/watch?v=uvWhSYBkZJ0). 
+[Training](https://carpentries.github.io/maintainer-onboarding/) on lesson maintenance workflows and [GitHub skill up sessions](https://carpentries.github.io/github-skill-up-maintainers/) will be offered in multiple time zones during onboarding to ensure you have the skills you need to succeed.
+
+The Maintainer community has a dedicated Slack channel, TopicBox email list, and monthly meetings to help support you and the lesson you are working on. 
+Lesson maintenance is asynchronous and can take place wherever it fits in your schedule, although occasional co-working sessions may be organised on Zoom to help work through challenges.
+
+The onboarding, skill-ups, and regular opportunities to connect with and learn from an established community make this a fantastic opportunity to develop your skills and get involved in Open Source in a supportive environment.
+
+## Which lessons need new Maintainers?
+
+We are recruiting Maintainers for all of our official lessons, including both English and Spanish language materials. 
+Although Maintainer Onboarding will be held in English, we strongly encourage applications from folks who are interested in maintaining one of the four official Carpentries lessons currently offered in Spanish.
+
+We are particularly in need of new Maintainers for the following lessons:
+
+* Data Carpentry:  
+  * [Foundations of Astronomical Data Science](https://datacarpentry.org/astronomy-python/)
+  * [Introduction to Cloud Computing for Genomics](https://datacarpentry.org/cloud-genomics/)
+  * [Data Analysis and Visualization in Python for Ecologists](https://datacarpentry.org/python-ecology-lesson/) (see note below)  
+  * [Data Analysis and Visualization in R for Ecologists](https://datacarpentry.org/R-ecology-lesson/)
+  * [Introduction to the Command Line for Genomics](https://datacarpentry.org/shell-genomics/)
+  * [Data Organization in Spreadsheets for Social Scientists](https://datacarpentry.org/spreadsheets-socialsci/)
+* Library Carpentry:  
+  * [Data Intro for Archivists](https://librarycarpentry.org/lc-data-intro-archives/)
+  * [Tidy Data for Librarians](https://librarycarpentry.org/lc-spreadsheets/)
+  * [Wikidata](https://librarycarpentry.github.io/lc-wikidata/)
+* Software Carpentry:  
+  * [The Unix Shell](https://swcarpentry.github.io/shell-novice/)
+  * [Databases and SQL](https://swcarpentry.github.io/sql-novice-survey/)
+
```

Comment:
@tobyhodges can we add for LC:

* [Introduction to Python](https://librarycarpentry.github.io/lc-python-intro/)
* [Introduction to R](https://librarycarpentry.github.io/lc-r/)
* [Introducing Computational Thinking](https://librarycarpentry.github.io/lc-computational-thinking/)